Automattic/vip-workflow-plugin
# Migrate custom status to use term_meta

## Description

- Migrate custom status to use `term_meta`, similar to #41
- Change the taxonomy used by custom status to be unique, so it won't interfere with Edit Flow. The slug has been left as is, just to not make it a minefield of bugs. This should be fine for now imo.
- Add in support for storing editorial metadata fields on a status so we can make that required in another PR
- Update the frontend of a status to show this new restriction.

## Testing

- Remove your custom status options as the taxonomy is different, so that should be sufficient.
- Ensure normal CRUD operations against a status works fine
- Ensure CRUD operations against an EM on a status works fine as well
